src/app/components/sidenav/sidenav.component.html

```diff
@@ -7,23 +7,23 @@
     <div *ngFor="let group of groups; trackBy: trackById" class="section" [ngClass]="'section--' + mode">
       <ng-container *ngSwitchCase="'expanded'">
         <mat-list-item *ngFor="let item of group.items; trackBy: trackById">
-          <ng-container *ngIf="isLoggedIn(item.provider) && !item.component">
+          <ng-container *ngIf="!item.component">
             <app-expand-menu [item]="item"></app-expand-menu>
           </ng-container>
 
-          <ng-container *ngIf="isLoggedIn(item.provider) && item.component">
+          <ng-container *ngIf="item.component">
             <adf-dynamic-component [data]="{ item: item, state: 'expanded' }" [id]="item.component"></adf-dynamic-component>
           </ng-container>
         </mat-list-item>
       </ng-container>
 
       <ng-container *ngSwitchCase="'collapsed'">
         <div class="list-item" *ngFor="let item of group.items; trackBy: trackById">
-          <ng-container *ngIf="isLoggedIn(item.provider) && !item.component">
+          <ng-container *ngIf="!item.component">
             <app-button-menu [item]="item"></app-button-menu>
           </ng-container>
 
-          <ng-container *ngIf="isLoggedIn(item.provider) && item.component">
```

Comment:
I don't think this modify will be ok when APS is not installed with ADW

Reply:
I feel the "LoggedIn" method is not required here, we already have custom rule to check same behaviour.
https://github.com/Alfresco/alfresco-digital-workspace-app/blob/develop/libs/content-ee/process-services-cloud-extension/assets/process-services-cloud.extension.json#L147

Reply:
Alright